libkdump/libkdump.c

```diff
@@ -10,6 +10,7 @@
 #include <stdarg.h>
 #include <stdlib.h>
 #include <unistd.h>
+#include <x86intrin.h>
```

Comment:
This file is quite new:
```commit 4542038bb61e37e63fb1d5bbcbef760f70005b29
Author: dwarak <dwarak@138bc75d-0d04-0410-961f-82ee72b054a4>
Date:   Fri Dec 12 16:53:27 2008 +0000
```

Please directly include used `*intrin.h` files instead.

Reply:
Are we saying 2008 is quite new? :) That's GCC 4.3-era.

Reply:
2008.. 2018... Same same but different